build-system/tasks/make-extension/template/shared/extensions/amp-__component_name_hyphenated__/__component_version__/storybook/Basic.amp.js

```diff
@@ -1,32 +1,29 @@
 import * as Preact from '#preact';
 import {withAmp} from '@ampproject/storybook-addon';
-import {withKnobs} from '@storybook/addon-knobs';
 
 export default {
   title: 'amp-__component_name_hyphenated__-__component_version_snakecase__',
-  decorators: [withKnobs, withAmp],
-
+  decorators: [withAmp],
   parameters: {
     extensions: [
       {name: 'amp-__component_name_hyphenated__', version: '__component_version__'},
     ],
     __storybook_experiments_do_not_add_trailing_comma__
   },
+  args: {
+    'data-example-property': 'example string property argument'
+  }
 };
 
 // __do_not_submit__: This is example code only.
-export const ExampleUseCase = () => {
+export const _default = (args) => {
   return (
     <amp-__component_name_hyphenated__
       width="300"
       height="200"
-      example-property="example string property value"
+      {...args}
     >
       This text is inside.
     </amp-__component_name_hyphenated__>
   );
 };
-
-ExampleUseCase.story = {
-  name: 'Example use case story'
```

Comment:
FMI: do we lose out on the name?

Reply:
Renders as `Example Use Case`. It's nicer to not specify it if it's they're equal, or nearly so.